website/sidebars.json

```diff
@@ -61,6 +61,10 @@
                 "label": "JavaScript API",
                 "ids": ["cicero-api", "ergo-api", "concerto-api"]
             }
+        ],
+        "FAQ": [
+            "for-lawyers",
```

Comment:
those should be consistent with the tag used to name the corresponding file (I think `faq-lawyer` and `faq-developer`)